app/src/atoms/InlineNotification/index.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react'
+import {
+  Icon,
+  JUSTIFY_SPACE_BETWEEN,
+  IconProps,
+  Flex,
+  DIRECTION_ROW,
+  ALIGN_CENTER,
+  COLORS,
+  SPACING,
+  TYPOGRAPHY,
+  BORDERS,
+  Btn,
+} from '@opentrons/components'
+import { StyledText } from '../text'
+
+import type { StyleProps } from '@opentrons/components'
+
+export type InlineNotificationType = 'alert' | 'error' | 'neutral' | 'success'
```

Comment:
Do we need to export the type?
```suggestion
 type InlineNotificationType = 'alert' | 'error' | 'neutral' | 'success'
```